src/swarm-tui/Swarm/TUI/View.hs

```diff
@@ -146,6 +146,9 @@ import Text.Printf
 import Text.Wrap
 import Witch (into)
 
+-- | The 2nd case is for key commands that have more than one btn in the ui
+type KeyCmd = Either (KeyHighlight, Text, Text) (KeyHighlight, [(Text, Text)], Text)
```

Comment:
Rather than having a type synonym for `Either`, I'd rather have an actual `data` type with two constructors like `SingleButton` and `MultiButton` or something descriptive like that.  (Then we can also get rid of the tuples at the same time.)  When reading things like `Just (Left (...))` in the code it's very hard to know what is going on; something like `Just (SingleButton (...))` would be easier to read and easier to track down the purpose.

Reply:
Agreed!  I'll make that change.